js/debug-bar-queries.dev.js

```diff
@@ -0,0 +1,42 @@
+(function(jQuery){
+	jQuery(document).ready( function(){
+
+		var elapsedSort = function(a, b){
+			// Account for elapsed times containing commas.
+			parsedA = parseFloat(a.dataset.ms_elapsed.replace(/,/g, ''));
+			parsedB = parseFloat(b.dataset.ms_elapsed.replace(/,/g, ''));
+
+			if(parsedA < parsedB) {
+				return 1;
+			}
+			if(parsedA > parsedB) {
+				return -1;
+			}
+			return 0;
+		}
+
+		var ordinalSort = function(a, b){
+			parsedA = parseInt(a.dataset.count.replace(/,/g, ''));
+			parsedB = parseInt(b.dataset.count.replace(/,/g, ''));
+
+			if(parseInt(a.dataset.count) > parseInt(b.dataset.count)) {
```

Comment:
I presume you meant to compare parsedA with parsedB here and in the next condition?
If not, you can remove the parseInt statements above this line.

I'd also suggest adding a helper method for the actual compares to get rid of the duplicate code.